src/utils/test/flatten.test.js

```diff
@@ -99,4 +102,22 @@ describe('flatten', () => {
     expect(flatten(['foo', func], { bool: true })).toEqual(['foo', 'static', 'bar']);
     expect(flatten(['foo', func], { bool: false })).toEqual(['foo', 'static', 'baz']);
   });
+  it('warns to refer styled-components', () => {
```

Comment:
nit: can you change this test name to "warns if trying to interpolate a normal React component"